include/aggregation_options.h

```diff
@@ -31,15 +31,18 @@ typedef std::unordered_map<std::string,
     MetricKindMap;
 
 struct QuotaAggregationOptions {
-  QuotaAggregationOptions() : num_entries(10000), refresh_interval_ms(1000) {}
+  QuotaAggregationOptions() : num_entries(10000), refresh_interval_ms(1000),
+      expiration_interval_ms(600000){}
 
   // Constructor.
   // cache_entries is the maximum number of cache entries that can be kept in
   // the aggregation cache. Cache is disabled when cache_entries <= 0.
   // refresh_interval_ms is the maximum milliseconds before an aggregated quota
   // request needs to send to remote server again.
-  QuotaAggregationOptions(int cache_entries, int refresh_interval_ms)
-      : num_entries(cache_entries), refresh_interval_ms(refresh_interval_ms) {}
+  QuotaAggregationOptions(int cache_entries, int refresh_interval_ms,
```

Comment:
Please add comment saying expiration_interval should be big.  It should be at lease 10 time bigger than rate limit window.